### DISCUSSION TITLE
Add a contact button somewhere?

### DISCUSSION THREAD

**Discussion by stan2544** [BrickTwo/BrickHunter#122] [Ideas]
Had to make an account here cause I couldn't find an email anywhere :D

Also they are merging BaP and PaB together: https://jaysbrickblog.com/news/introducing-legos-new-pick-and-build-hub-bricks-pieces-to-merge-into-pick-a-brick-and-build-a-minifigure-available-online/

**Comment by BrickTwo** [comment]
there is a contact info in the BrickHunter page in the Chrome Extension Store :) Anyway, there is already planed to have something in the next Major Version of BrickHunter.

Yes, I'm already aware the the mergin of BaP and PaB. Since they just merge to PaB, I think BrickHunter should still work for the order process until I will publish an update. The single parts section though will sadly not work for some time until I have adjusted everything necessary.
